jupyter_server/services/contents/largefilemanager.py

```diff
@@ -31,7 +32,7 @@ def save(self, model, path=""):
                 raise web.HTTPError(400, "No file content provided")
 
             os_path = self._get_os_path(path)
-            self.log.debug("Saving %s", os_path)
+            self.log.debug(f"Saving chunk {chunk} of file {os_path}")
```

Comment:
What does this typically look like in the logs for large and small files? 1, [2, 3, ...], -1 ? Should we replace "chunk -1" with "last chunk" ?

Reply:
I don't know, I was trying to write a big file with JupyterLab but it doesn't seem to send `chunk` in the model.

Reply:
Yeah, I'm seeing the same thing.  Chunk size appears to default to 100KB and I'm writing a 12MB file and no `chunk` field is in the model.  I wonder if "chunks" are a Notebook frontend thing (or folks with their own frontends)?

Hmm, the save method will defer to the superclass when no chunking is specified and superclass (`FileContentsManager`) will call the save hooks, so at least they're still being called.  As a result, the severity of this is dramatically lessened, although I'm curious about the history of "Chunks and the LargeFileManager".

Reply:
I tried with Notebook, but there's not `chunk` there either. I'll open an issue in Jupyter Lab.

Reply:
So it turns out JupyterLab uses chunks when dragging and dropping files in the file browser, and this is the debug logs:
```
[I 2022-03-11 11:56:37.828 ServerApp] Uploading file to /test.txt
[D 2022-03-11 11:56:37.828 ServerApp] Saving chunk 1 of file /home/david/github/davidbrochart/jupyter_server/test.txt
[D 2022-03-11 11:56:37.846 ServerApp] 201 PUT /api/contents/test.txt?1646996197800 (::1) 29.02ms
[D 2022-03-11 11:56:37.889 ServerApp] Saving chunk 2 of file /home/david/github/davidbrochart/jupyter_server/test.txt
[D 2022-03-11 11:56:37.893 ServerApp] 200 PUT /api/contents/test.txt?1646996197865 (::1) 11.90ms
...
[I 2022-03-11 11:56:41.764 ServerApp] Saving file at /test.txt
[D 2022-03-11 11:56:41.764 ServerApp] Saving chunk -1 of file /home/david/github/davidbrochart/jupyter_server/test.txt
[D 2022-03-11 11:56:41.766 ServerApp] 200 PUT /api/contents/test.txt?1646996201755 (::1) 5.04ms
```
@vidartf :+1: to replace `Saving chunk -1` with `Saving last chunk`.